packages/neos-ui/src/Containers/RightSideBar/Inspector/index.js

```diff
@@ -277,12 +281,21 @@ export default class Inspector extends PureComponent {
                         // Render each tab as a TabPanel
                         //
                         .map(tab => {
+                            const notifications = validationErrors ?
+                                tab.groups.reduce((notifications, group) => {
+                                    return group.items.reduce((notifications, item) => (
+                                        Object.keys(validationErrors).includes(item.id) ? notifications + 1 : notifications
+                                    ), 0);
+                                }, 0) : 0;
+
                             return (
                                 <TabPanel
                                     key={$get('id', tab)}
                                     id={$get('id', tab)}
                                     icon={$get('icon', tab)}
                                     groups={$get('groups', tab)}
+                                    notifications={notifications}
+                                    title={Boolean(notifications) && <Badge className={style.tabs__notificationBadge} label={String(notifications)}/>}
```

Comment:
Could we maybe have a more informative label for notifications? E.g. "3 properties in this tab are failing validation". That would require I18n ofc.

Reply:
Sure will take care